src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -166,6 +166,44 @@ class SeekBar extends Slider {
   stepBack() {
     this.player_.currentTime(this.player_.currentTime() - STEP_SECONDS);
   }
+
+  /**
+   * Toggles the playback state of the player
+   * This gets called when enter or space is used on the seekbar
+   *
+   * @param {EventTarget~Event} event
+   *        The `keydown` event that caused this function to be called
+   *
+   */
+  handleAction(event) {
+    if (this.player_.paused()) {
+      this.player_.play();
+    } else {
+      this.player_.pause();
+    }
+  }
+
+  /**
+   * Called when this SeekBar has focus and a key gets pressed down. By
+   * default it will call `this.handleAction` when the key is space or enter.
+   *
+   * @param {EventTarget~Event} event
+   *        The `keydown` event that caused this function to be called.
+   *
+   * @listens keydown
+   */
+  handleKeyPress(event) {
+
+    // Support Space (32) or Enter (13) key operation to fire a click event
+    if (event.which === 32 || event.which === 13) {
+      event.preventDefault();
+      this.handleAction(event);
+    } else if (super.handleKeyPress) {
+
+      // Pass keypress handling up for unsupported keys
+      super.handleKeyPress(event);
```

Comment:
My thought on this was that unhandled keypresses by each control bar element would bubble up here to higher components; for example, if Space isn't supported by a control bar component, it would bubble all the way up to the control bar itself, which would interpret it as 'toggle playback state'. That way, you wouldn't need to put this kind of handling into each sub-component, and the control bar itself could implement the full set of keyboard shortcuts that are becoming the de-facto standard for YouTube (Space, Up Arrow, Down Arrow, Home, End, 0..9, F, M, etc.) But I realize this isn't what's happening here - it's bubbling up to the component this extends, not its structural parent. So maybe the root `Component` needs to pass it up to its parent component, if it has one?